package.json

```diff
@@ -81,6 +85,7 @@
     "@types/express": "^4.17.13",
     "@types/jest": "^29.4.0",
     "@types/node": "^18.16",
+    "@types/react": "^18",
```

Comment:
Same here with the types for react.

Reply:
Same as above. Will confirm tomorrow if the install is still an issue.

Reply:
Have removed react as a dep and [all is well](https://github.com/prosopo/polkadot-testnet-faucet/pull/8).